### PR TITLE
My first verison of Springboot demo homework

### PR DESCRIPTION
**Pull request by ghost**
I made my first version of Springboot homework.
After the push i realized my gitignore is not set correctly... :)

### CODE REVIEW

File: Springboot-demo/src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+spring.application.name=springboot-demo
+spring.datasource.url=jdbc:mysql://localhost:3306/employee
+spring.datasource.username=root
+spring.datasource.password=root
```
Comment (lyancsie) [inline Springboot-demo/src/main/resources/application.properties:5]: i read your pr description, but you should never do this at all

File: Springboot-demo/src/main/java/com/hollszabolcs/demo/model/Employee.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    @Id
+    private int id;
+    private String name;
+    private int age;
+}
+
+
+
+
+
```
Comment (lyancsie) [inline Springboot-demo/src/main/java/com/hollszabolcs/demo/model/Employee.java:23]: as the famous song goes: why so many empty lines?

File: Springboot-demo/src/main/java/com/hollszabolcs/demo/controller/EmployeeController.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    @Autowired
+    private EmployeeService employeeService;
+
+
+    @GetMapping("/employee")
+    List<Employee> getEmployee() {
+        return employeeRepository.findAll();
+    }
+
+    @GetMapping("rin{name}")
```
Comment (lyancsie) [inline Springboot-demo/src/main/java/com/hollszabolcs/demo/controller/EmployeeController.java:26]: ???

File: Springboot-demo/src/main/java/com/hollszabolcs/demo/controller/EmployeeController.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    public String getUpperClassNames(@PathVariable(required = false) String name) {
+        if (name == null) {
+            return "No name provided";
+        }
+        String upperCaseName = employeeService.getUpperCaseName(name);
+        return "Upper case name: " + upperCaseName;
+    }
```
Comment (lyancsie) [inline Springboot-demo/src/main/java/com/hollszabolcs/demo/controller/EmployeeController.java:33]: Business logic should be in the service

File: Springboot-demo/src/main/java/com/hollszabolcs/demo/controller/EmployeeController.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class EmployeeController {
+    @Autowired
+    private EmployeeRepository employeeRepository;
```
Comment (lyancsie) [inline Springboot-demo/src/main/java/com/hollszabolcs/demo/controller/EmployeeController.java:16]: Never autowire repo in your controller layer - autowire only services

File: Springboot-demo/mvnw.cmd [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+if exist "%USERPROFILE%\mavenrc_post.bat" call "%USERPROFILE%\mavenrc_post.bat"
+if exist "%USERPROFILE%\mavenrc_post.cmd" call "%USERPROFILE%\mavenrc_post.cmd"
+:skipRcPost
+
+@REM pause the script if MAVEN_BATCH_PAUSE is set to 'on'
+if "%MAVEN_BATCH_PAUSE%"=="on" pause
+
+if "%MAVEN_TERMINATE_CMD%"=="on" exit %ERROR_CODE%
+
+cmd /C exit /B %ERROR_CODE%
```
Comment (lyancsie) [inline Springboot-demo/mvnw.cmd:205]: please do a research what this is and write it down here briefly if you did this way :)

File: Springboot-demo/src/main/resources/templates/hello-world.html [hunk 7]
```diff
@@ -0,0 +1,5 @@
+<head>
+</head>
+<body>
+<h1>Hello everyone</h1>
+</body>
```
Comment (lyancsie) [inline Springboot-demo/src/main/resources/templates/hello-world.html:5]: Why do you need this?

File: Springboot-demo/mvnw.cmd (hunk 6 above)
Comment (hollszabolcs) [reply Springboot-demo/mvnw.cmd:205]: I Don't have a clue what is this, and why was it here... :D
But some kind of script for maven build up?

**Pull request merged**